src/rtk/features/overviewOrg/overviewOrgSlice.ts

```diff
@@ -104,17 +135,34 @@ const createCopyOrg = (org: OverviewOrg) => {
   };
 };
 
-export const fetchOverviewOrgs = createAsyncThunk('overviewOrg/fetchOverviewOrgs', async () => {
-  return await axios
-    .get('https://jsonplaceholder.typicode.com/users')
-    .then((response) => response.data)
-    .catch((error) => console.log(error));
-});
+export const fetchOverviewOrgs = createAsyncThunk(
```

Comment:
Instead of doing it like this with a state. There should be own fetch-methods for inbound and outbound. This will make sure there's no need for an own layoutState in this slice. It's difficult to explain what my plan is, but basically I think it would be wise to split up everything that has to do with inbound and outbound in this slice to minimalize state.

Why should we have it like this you'll ask. Since we don't have a BFF yet, we have to use the slices in our frontend like a backend. The mapping of data is actually the backend's responsibility. The idea of splitting backend-endpoints as much as possible and be as specific as possible is related to restful approach. That's what my comments in this file is related to 😊

Reply:
In the future we hopefully will have a good restful backend and then we don't need to do the mapping in frontend anymore. When that happens the splitting we're doing now will pay off and it will be easy to transition to it.

Reply:
Agree 😊